### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by Pandaver** [MeteorDevelopment/meteor-client#2535]
### Describe the bug

crashed when start up.

### Steps to reproduce

```bash
jdk-17.0.3.7 from https://adoptium.net/
MulitMC 0.6.16-3431
Fabric Loader 0.14.8 install from MulitMC
Mod just enable Meteor Client

Click Launch And Crashed
log: https://paste.ee/p/etsuk
```

### Meteor Version

0.5.0-1538

### Minecraft Version

1.19

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by playeur242** [comment]
seems like a jdk issue, try downloading java18 and use it
also do you have any others mods ?

**Comment by pikacan360** [comment]
download multimc/polymc and set it up and see if it works, and if not could to say what the log says in multimc/polymc

**Comment by Pandaver** [comment]
> seems like a jdk issue, try downloading java18 and use it also do you have any others mods ?

I try, not work

**Comment by MineGame159** [comment]
Fixed in 9a35451c9e7ad74b813731645c603c413e795b81.
